config/routes.rb

```diff
@@ -24,7 +24,9 @@
       resources :tasks, only: %i(index show) do
         resource :output, only: :show
       end
-      resources :deploys, only: %i(index create)
+      resources :deploys, only: %i(index create) do
+        post 'create_status'
```

Comment:
I'm not a fan of that endpoint. I'm no REST extremist, but it's not very consistent with the rest of the API.

A better API IMO would be:

`POST /.../deploys/:id/release_statuses` with `state = healthy / faulty`.

So in term of routes:

```ruby
resources :release_statuses, only: %i(create)
```